validator/validator.go

```diff
@@ -0,0 +1,61 @@
+// Package validator usage:
+
+// type Application struct {
+// 	Name      string `json:"name" validate:"required,printascii"`
+// 	Namespace string `json:"namespace"`
+// }
+
+// Definitions: []definition.Definition{
+// 	{
+// 		Method:   definition.Create,
+// 		Function: Handle,
+// 		Parameters: []definition.Parameter{
+// 			{
+// 				Source:    definition.Query,
+// 				Name:      "target1",
+// 				Operators: []definition.Operator{validator.Var("gt=0,lt=10")},
+// 			},
+// 			{
+// 				Source: definition.Body,
+// 				Name:   "app",
+// 				// Should we call this automatically?
+// 				Operators: []definition.Operator{validator.Struct()},
+// 			},
+// 		},
+// 	},
+// },
+
+package validator
+
+import (
+	"context"
+
+	"gopkg.in/go-playground/validator.v9"
+)
+
+var std = validator.New()
```

Comment:
Please add documentation to point to std validation and mention that only std is supported now.

This can be done in another PR.

For example (or documentation), we need one for single variable and one for struct.

Reply:
OK, will add it in another README PR.